### DISCUSSION TITLE
Add option to set "min", "max" and "step" values for sliders

### DISCUSSION THREAD

**Discussion by infinicode** [Clooos/Bubble-Card#903] [Feature requests]
Hello. First of all, thanks for this great set of cards!

I would like to kindly request to add some new options to the sliders configuration:
- "min": minimum value of slider
- "max": maximum value of slider, really helpful for speakers
- "step": step size for the slider, for example, to set covers in steps of 10%.

This request is inspired in the configuration options of the [Slider Entity Row card](https://github.com/thomasloven/lovelace-slider-entity-row)

Thanks in advance!

**Comment by Clooos** [comment]
Hi! This will finally be a feature in the next release! 😃

**Comment by infinicode** [comment]
Oh this is great! Thanks @Clooos !!!!
